### PR TITLE
implements the issues #1, #2, #3 and #5

### PR DESCRIPTION
**Pull request by ChristianHirsch**
@tosmi: hera are my code changes wich handles all the issues which I opened. on Ubuntu 14.04 it installs reaktor from scratch without any erros. this has not worked for me before. I also updated the docs, but there are still some parts missing.. especially for systems which are not Ubuntu ;)

### CODE REVIEW

**Comment by ChristianHirsch** [comment]
@tosmi ok, so I also created the possibility to handle the masters.txt file.

**Comment by tosmi** [comment]
wow, thanks for all the work. looks good to me, there's just one thing which i would like to see changed: could you move parameters that do not depend on a conditional from params.pp to init.pp? i find it way easier to understand a module if simple defaults are included in params.pp... i will leave a comment in the pull request do clearify the issue.

File: manifests/params.pp [hunk 1]
```diff
@@ -4,14 +4,49 @@
 # It sets variables according to platform.
 #
 class reaktor::params {
+
+  $ensure       = 'present'
```
Comment (tosmi) [inline manifests/params.pp]: that's a simple default without any conditional, this could be moved to init.pp

File: manifests/service.pp [hunk 2]
```diff
@@ -5,10 +5,53 @@
...
+  $default_config = {
+    'RACK_ROOT'                   => $dir,
+    'RESQUE_WORKER_USER'          => $user,
+    'RESQUE_WORKER_GROUP'         => $group,
+    'REAKTOR_PUPPET_MASTERS_FILE' => "${dir}/masters.txt",
+  }
+  $config = merge($default_config, $::reaktor::config)
+
+  file { $init_file_reaktor_redis:
+    ensure  => $ensure,
```
Comment (tosmi) [inline manifests/service.pp:40]: hm, i'm a blind? i can't seem to find the definition of $ensure in class reaktor::service.

File: manifests/params.pp (hunk 1 above)
Comment (tosmi) [inline manifests/params.pp]: also i can't seem to find the usage of reaktor::params::ensure in init.pp...

File: manifests/params.pp [hunk 3]
```diff
@@ -4,14 +4,49 @@
...
+  $servers              = 1
+  $max_conns            = 1024
+  $max_persistent_conns = 512
+  $timeout              = 30
+  $environment          = 'production'
+  $pid                  = 'tmp/pids/reaktor.pid'
+  $log                  = 'reaktor.log'
+
+  # masters.txt file
+  $manage_masters = true
```
Comment (tosmi) [inline manifests/params.pp:32]: the defaults from line 10 to 34 could be moved to init.pp as they do not depend a condition

**Comment by ChristianHirsch** [comment]
@tosmi I removed the ensure parameter because it does not make any sense in my opinion. I just forgot to remove it also in the params.pp

**Comment by tosmi** [comment]
thanks once again for your work. currently it's a little bit hard for me to find the time to work on this module, but it's on my todo list.

**Pull request merged**